### PR TITLE
Added Test_parseMycnf, Test_scrapeTableStat.

### PR DESCRIPTION
**Pull request by roman-vynar**
```
weber@pluto:~/git/mysqld_exporter (more-tests)$ go test -race -v
=== RUN   Test_scrapeTableStat

  Counters comparison ✔✔✔✔✔✔✔✔✔


9 total assertions

--- PASS: Test_scrapeTableStat (0.00s)
=== RUN   Test_scrapeQueryResponseTime

  Counters comparison ✔✔✔✔✔✔✔✔✔✔✔✔✔✔


23 total assertions


  Histogram comparison ✔


24 total assertions

--- PASS: Test_scrapeQueryResponseTime (0.00s)
=== RUN   Test_parseMycnf

  Various .my.cnf configurations 
    Local tcp connection ✔
    Local tcp connection on non-default port ✔
    Socket connection ✔
    Socket connection ignoring defined host ✔
    Remote connection ✔
    Missed user ✔
    Missed password ✔
    No [client] section ✔
    Invalid config ✔


33 total assertions

--- PASS: Test_parseMycnf (0.00s)
PASS
ok  	_/Users/weber/git/mysqld_exporter	1.034s
```

### CODE REVIEW

**Comment by SuperQ** [comment]
It looks like you also made some changes to the histogram test.  Please update the change description to include this.

**Comment by roman-vynar** [comment]
Do you mean PR title? Updated it.

**Comment by SuperQ** [comment]
Generally it's nice to keep commit/PR titles short (80 char) and include more information in the body.

For example here's a reasonable git commit log entry:
https://github.com/prometheus/prometheus/commit/66767121ab5e5cdf12209d727551f8a80708ec73

**Comment by roman-vynar** [comment]
Done.

**Comment by SuperQ** [comment]
:+1:

**Comment by roman-vynar** [comment]
Added one more test - Test_scrapeTableStat.

File: mysqld_exporter_test.go [hunk 1]
```diff
@@ -55,31 +111,32 @@ func Test_scrapeQueryResponseTime(t *testing.T) {
...
-		"10":     1.5773549999999998,
-		"100":    1.5773549999999998,
-		"1000":   1.5773549999999998,
-		"10000":  1.5773549999999998,
-		"100000": 1.5773549999999998,
-		"1e+06":  1.5773549999999998,
-		"+Inf":   1.5773549999999998,
+	// Test counters
+	expectTimes := []CounterResult{
+		{LabelMap{"le": "1e-06"}, 0},
```
Comment (fabxc) [inline mysqld_exporter_test.go]: Please avoid anonymous struct initializations.

Comment (roman-vynar) [inline mysqld_exporter_test.go]: @fabxc is this what you mean (the last commit)?

**Comment by roman-vynar** [comment]
up :)

**Comment by SuperQ** [comment]
:+1: Thanks for more tests

**Pull request merged**